tests/common/fixtures/advanced_reboot.py

```diff
@@ -481,6 +486,11 @@ def __fetchTestLogs(self, rebootOper=None):
                 {'src': syslogFile, 'dest': log_dir, 'flat': True},
                 {'src': sairedisRec, 'dest': log_dir, 'flat': True},
                 {'src': swssRec, 'dest': log_dir, 'flat': True},
+                # Logs from before reboot
+                {'src': '/host/syslog.99', 'dest': log_dir_before_reboot, 'flat': True, 'fail_on_missing': True},
```

Comment:
Let's not fail_on_missing. These files may be missing for whatever reason. There are cases when swss.rec, bgp.log is missing, and we will fail the testcase, by being too strict.